terraform/aks/config/test.tfvars.json

```diff
@@ -0,0 +1,20 @@
+{
+  "cluster": "test",
+  "cluster_rg": "s189t01-tsc-ts-rg",
+  "cluster_name": "s189t01-tsc-test-aks",
+  "namespace": "tra-development",
+  "environment": "test",
+  "deploy_azure_backing_services": false,
```

Comment:
Normally this is only false for review apps. Permanent apps use an azure backing service